maven-slf4j-wrapper/src/main/java/org/apache/maven/logwrapper/LogLevelRecorder.java

```diff
@@ -42,14 +42,7 @@
 
     public LogLevelRecorder( String threshold )
     {
-        Level level = determineThresholdLevel( threshold );
-
-        if ( level.toInt() < Level.WARN.toInt() )
-        {
-            throw new IllegalArgumentException( "Logging severity thresholds can only be set to WARN or ERROR" );
-        }
-
-        logThreshold = level;
+        logThreshold = determineThresholdLevel( threshold );
```

Comment:
We only accept this two level logging https://github.com/apache/maven/pull/337/files#diff-1539b34a819f425067138253cfac27adR35
And the logging level value is `   ERROR(40, "ERROR"),
    WARN(30, "WARN"),
    INFO(20, "INFO"),
    DEBUG(10, "DEBUG"),
    TRACE(0, "TRACE");`